examples/keras_recipes/md/float8_training_and_inference_with_transformer.md

```diff
@@ -53,7 +55,7 @@ Note: The dependency on TensorFlow is only required for data processing.
 
 
 ```python
```

Comment:
What is going on with this block? I would not expect it to render in this form

Reply:
![image](https://github.com/keras-team/keras-io/assets/20734616/7dd59c90-c220-437f-9fa9-a0d7a7d9c635)

The list doesn't render correctly.
The format is referenced from this one:
https://github.com/keras-team/keras-io/blob/master/examples/nlp/parameter_efficient_finetuning_of_gpt2_with_lora.py#L314-L324